device-connectors/src/testflinger_device_connectors/devices/zapper_kvm/__init__.py

```diff
@@ -77,6 +101,17 @@ def _validate_configuration(
             )
             retries = self.job_data["provision_data"].get("robot_retries", 1)
 
+        # If a SSO email is specified, e.g. UC, username must match
+        # the local-part because that would be the only user available
+        # on the DUT after provisioning.
```

Comment:
I don't think this is correct. The username is the launchpad user associated with that sso login and isn't necessarily the same as what comes before the "@" in the email. For example, Mine is `paul.larson@...` for the email, but the resulting user created in Ubuntu Core is `pwlars`

Reply:
oh, TIL. I think I learned from the wrong example then (ce-certification-qa). So we can't do much except for making it clear in the documentation.